docs/source/new-project.rst

```diff
@@ -1,15 +1,14 @@
 .. testsetup:: *
 
-    from pytorch_lightning.core.lightning import LightningModule
-    from pytorch_lightning.core.datamodule import LightningDataModule
-    from pytorch_lightning.trainer.trainer import Trainer
     import os
     import torch
     from torch.nn import functional as F
     from torch.utils.data import DataLoader
-    from torch.utils.data import DataLoader
-    import pytorch_lightning as pl
     from torch.utils.data import random_split
+    import pytorch_lightning as pl
```

Comment:
move it here:
https://github.com/PyTorchLightning/pytorch-lightning/blob/f677efe61ecf28af1e5216a5baf502be6f394564/docs/source/conf.py#L357

Reply:
all the imports or all lightning imports or just `import pytorch_lightning as pl`??

Reply:
we can include both in the config

Reply:
well, maybe keep it there as users want to do just copy-paste...